### PR TITLE
Change daily job to use `swift-5.0-branch`

### PR DESCRIPTION
**Pull request by norio-nomura**

### CODE REVIEW

**Comment by norio-nomura** [comment]
Filed [SR-9454](https://bugs.swift.org/browse/SR-9454)

File: Tests/YamsTests/TestHelper.swift [hunk 1]
```diff
@@ -12,6 +12,29 @@ import CDispatch
 import Foundation
 import XCTest
 
+/// https://bugs.swift.org/browse/SR-9454
+func doesSR9454Affect(to name: String = #function) -> Bool {
+#if swift(>=4.1.50)
+#if compiler(>=5) && os(Linux)
+    let string = "LINE1_67あ\nLINE2_7890\nLINE3_8901\n"
+    let rangeOfFirstLine = string.lineRange(for: string.startIndex..<string.startIndex)
```
[the remark this answers is not in the document]
Comment (norio-nomura) [reply Tests/YamsTests/TestHelper.swift]: Yes, this is intentional.
It comes from [`StringTests.testConfirmBehaviorOfStandardLibraryAPI()`](https://github.com/jpsim/Yams/pull/139/files#diff-24557e19a84885d8da5f5da1c7532eb8R23)

File: .circleci/config.yml [hunk 2]
```diff
@@ -91,19 +91,29 @@ jobs:
...
+        type: string
     macos:
       xcode: << parameters.xcode >>
+    environment:
+      SWIFTPM_FLAGS: << parameters.swiftpm_flags >>
     steps: &steps-for-swiftpm
       - checkout
-      - run: swift test
-      - run: swift test -Xswiftc -DUSE_UTF8
+      - run: swift test ${SWIFTPM_FLAGS}
```
[the remark this answers is not in the document]
Comment (norio-nomura) [reply .circleci/config.yml]: I'll try. 👍

[the remark this answers is not in the document]
Comment (norio-nomura) [reply .circleci/config.yml]: As far as [the processed config](https://circleci.com/gh/jpsim/Yams/2092#config/containers/0) is seen, it is likely to be able to pass parameters directly.
However, it seems that ["Your 14 day macOS trial has ended. Please go to the macOS plan page and select a plan to continue building."](https://circleci.com/gh/jpsim/Yams/2092#usage-queue/containers/0) message is displayed and the job is not executed.

**Comment by jpsim** [comment]
> However, it seems that ["Your 14 day macOS trial has ended. Please go to the macOS plan page and select a plan to continue building."](https://circleci.com/gh/jpsim/Yams/2092#usage-queue/containers/0) message is displayed and the job is not executed.

🤦‍♂️

I've had nothing but bad luck with Circle CI and their OSS macOS plan. Literally a year and a half of emailing back and forth with them, them misconfiguring my projects.

This has gotten so ridiculous, which was another one of my motivations for giving Azure Pipelines serious consideration. I'm continuing to try to move to Azure Pipelines again in #140.

**Pull request closed without merging**